tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -430,12 +447,18 @@ else if (args[i].toLowerCase().equals("-help")) {
            printUsage();
            System.exit(0);
```

Comment:
How about consistently existing with `System.exit(0)` here and above if no arguments are given?

Reply:
The shell tools I've tried that accept mandatory file arguments like `file`, `grep`, `chmod`, `rm`, `mv`, `cp` all exit with non-0 exit codes when they're given no arguments, I tried to be consistent with that. If you would still prefer to exit with `System.exit(0)` I can change it.

Reply:
Hmm, I'd rather put my time in something less administrative :-), let me know if you want me to change some exit values in this PR still.

Reply:
Fine by me as long as commits are focused on small, specific changes.

Reply:
I think you misunderstood what I meant, I'd rather not to this in this PR, please let me know if I need to change exit values of SANY somewhere to get this PR merged.

Reply:
No :-)